transaction/context.go

```diff
@@ -0,0 +1,45 @@
+package transaction
+
+import (
+	"context"
+	"fmt"
+)
+
+type contextKey int
+
+const (
+	trackedKey contextKey = iota
+	transactionIDKey
```

Comment:
Can you elaborate more on the problems with the imports? Is it only about package naming? Go and name the packages to whatever you want though, but besides renaming I do not see the value in here.